gbpservice/nfp/configurator/drivers/loadbalancer/v1/haproxy/haproxy_lb_driver.py

```diff
@@ -84,7 +84,9 @@ def configure_interfaces(self, context, resource_data):
                 msg = ("Failed to configure log forwarding for service at %s. "
                        "Error: %s" % (mgmt_ip, err))
                 LOG.error(msg)
-                return result_log_forward
+                # Temporary workaround to prevent log forward API
```

Comment:
Lets remove this comment. Lets put the following comment.

Failure in log forwarding configuration will not block chain creation. Error is logged for detecting failure.